pkgs/development/python-modules/pytorch/bin.nix

```diff
@@ -19,7 +19,7 @@ let
   platform = if stdenv.isDarwin then "darwin" else "linux";
   srcs = import ./binary-hashes.nix version;
   unsupported = throw "Unsupported system";
```

Comment:
This variable is only used once and thus could be inlined.

Reply:
Well, there are many variables only used once, and the line where it is used would get a bit long/tedious if it is inlined.

Reply:
ok.